change/@fluentui-react-charting-880075c2-33de-4bf3-b00a-d31bc76baf6d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Upgrading d3-scale dependent libraries",
+  "packageName": "@fluentui/react-charting",
```

Comment:
Mention the steps that you performed to upgrade the packages and deduplicate the yarn.lock file in the PR description.
Also mention the validations that you checked for.

Reply:
Done